client/dashboard/sites/settings-subscription-gifting/index.tsx

```diff
@@ -17,7 +27,7 @@ const fields: Field< SiteSettings >[] = [
 		Edit: ( { field, onChange, data, hideLabelFromVision } ) => {
 			const { id, getValue } = field;
 			return (
-				<ToggleControl
+				<CheckboxControl
```

Comment:
It would be better to add checkbox controls to dataform 🙂

Reply:
I'll open a follow-up issue 👍

Reply:
DOTCOM-13267